src/constraints/storage_constraints.jl

```diff
@@ -99,6 +107,32 @@ function add_elec_storage_dispatch_constraints(m, p, b; _n="")
             sum(m[Symbol("dvStorageEnergy"*_n)][b])
         )
     end
+
+    if p.s.storage.attr[b].is_ldes
+        @constraint(m, m[Symbol("dvStoragePower"*_n)][b] == m[Symbol("dvStorageEnergy"*_n)][b] / p.s.storage.attr[b].duration)
+    end
+
+    if p.s.storage.attr[b].require_start_and_end_charge_to_be_equivalent
+        # @constraint(m, m[:dvStoredEnergy]["ElectricStorage",maximum(p.time_steps)] == p.s.storage.attr[b].soc_init_fraction * m[Symbol("dvStorageEnergy"*_n)][b] )
+        print("\nSetting init SOC = final SOC\n")
+        @constraint(m,
+            m[Symbol("dvStoredEnergy"*_n)][b, 0] == m[:dvStoredEnergy]["ElectricStorage", maximum(p.time_steps)]
+        )
+    end
+
+    # Prevent charging and discharging of the battery at the same time
```

Comment:
would to be good to make this an input option (like ElectricUtility.allow_simultaneous_export_import) so can avoid the 8760 bin if want/need to.

Reply:
done